src/main/java/com/zendesk/maxwell/schema/SchemaCapturer.java

```diff
@@ -109,6 +109,14 @@ private void captureTable(Table t) throws SQLException {
 			String colEnc     = r.getString("CHARACTER_SET_NAME");
 			int colPos        = r.getInt("ORDINAL_POSITION") - 1;
 			boolean colSigned = !r.getString("COLUMN_TYPE").matches(".* unsigned$");
+			Long columnLength;
+
+			// Only available for datetime/timestamp/time
+			try {
+				columnLength = r.getLong("DATETIME_PRECISION");
+			} catch ( java.sql.SQLException e ) {
+				columnLength = null;
```

Comment:
Actually `r` is not null, we have the row but without the column. That's why it does:
`java.sql.SQLException: Column 'DATETIME_PRECISION' not found.`